gemini/gemini_main.py

```diff
@@ -898,6 +898,11 @@ def lof_interactions_fn(parser, args):
                   recessive inheritance model')
     add_inheritance_args(parser_auto_rec, gt_ll=True)
 
+    parser_auto_rec.add_argument('--format',
+                    dest='format',
+                    default='default',
+                    help='Format of output (JSON or default)'
+                    )
```

Comment:
If all of the inheritance tools have this option, can you put this into the `add_inheritance_args` function?